packages/use-intl/test/react/useTranslations.test.tsx

```diff
@@ -1,9 +1,7 @@
 import {render, screen} from '@testing-library/react';
 import {parseISO} from 'date-fns';
-// eslint-disable-next-line import/no-named-as-default -- False positive
```

Comment:
Not too bad!

<img width="763" alt="Screenshot 2023-12-06 at 17 51 22" src="https://github.com/amannn/next-intl/assets/4038316/71b19edd-4d7d-4a33-b453-1335b4debea4">